src/main/java/core/basesyntax/figures/Circle.java

```diff
@@ -0,0 +1,27 @@
+package core.basesyntax.figures;
+
+import core.basesyntax.Figure;
+import core.basesyntax.enums.FigureType;
+
+public class Circle extends Figure {
+    private static final double PI = 3.14;
+    private final double radius;
+    private final String color;
+
+    public Circle(double radius, String color) {
+        this.radius = radius;
+        this.color = color;
```

Comment:
for all other figures also
```suggestion
        super(color);
        this.radius = radius;
```